CONTRIBUTING.md

```diff
@@ -1,38 +1,24 @@
-# Contributing to the Kata Containers project
+# Contribute to the Kata Containers project
 
 * [Code of Conduct](#code-of-conduct)
+* [Pull requests](#pull-requests)
 * [Golang Coding Style](#golang-coding-style)
 * [Certificate of Origin](#certificate-of-origin)
-* [Pull requests](#pull-requests)
-    * [Before starting work on a PR](#before-starting-work-on-a-pr)
-    * [Before submitting a PR](#before-submitting-a-pr)
-    * [Normal PR workflow](#normal-pr-workflow)
-        * [First PR example](#first-pr-example)
-            * [Updating your PR based on review comments](#updating-your-pr-based-on-review-comments)
-    * [Temporarily blocking a PR](#temporarily-blocking-a-pr)
-    * [Assisted PR workflow](#assisted-pr-workflow)
-    * [Re-vendor PRs](#re-vendor-prs)
-    * [Stable branch backports](#stable-branch-backports)
+* [GitHub basic setup](#github-basic-setup)
+* [GitHub best practices](#github-best-practices)
+* [GitHub workflow](#github-workflow)
+* [Re-vendor PRs](#re-vendor-prs)
+* [Use static checks for validation](#use-static-checks-for-validation)
 * [Patch format](#patch-format)
-    * [General format](#general-format)
-    * [Subsystem](#subsystem)
-    * [Advice](#advice)
-    * [Verification](#verification)
-    * [Examples](#examples)
-        * [Main patch](#main-patch)
-        * [Supplementary patch](#supplementary-patch)
 * [Reviews](#reviews)
-    * [Review Examples](#review-examples)
 * [Continuous Integration](#continuous-integration)
 * [Contact](#contact)
 * [Project maintainers](#project-maintainers)
-* [Issue tracking](#issue-tracking)
-* [Closing issues](#closing-issues)
 
 The Kata Containers project is an open source project licensed under the
 [Apache License, Version 2.0](https://www.apache.org/licenses/LICENSE-2.0).
 
-It comprises a number of repositories under the [GitHub Kata
+It comprises a number of repositories under the [GitHub* Kata
```

Comment:
Nit: We tend not to use the asterisk in this context.

Reply:
Done.